.gitignore

```diff
@@ -11,3 +11,5 @@ npm-debug.log
 # coverage
 .nyc_output
 coverage
+
+.DS_Store
```

Comment:
Haha, yeh I have one setup just not on my personal laptop (don't use it much 👀 ) just thought it might be useful as I'm probably not the only one. Can remove if you want?